go/signedexchange/signedexchange.go

```diff
@@ -26,6 +27,14 @@ type Exchange struct {
 	payload []byte
```

Comment:
Would you mind exporting these fields as part of this change? Or I can open a PR if you prefer.